docs/content/en/docs/reference/baremetal/bare-preparation.md

```diff
@@ -5,3 +5,71 @@ weight: 20
 description: >
   Set up a Bare Metal cluster to prepare it for EKS Anywhere
 ---
+After gathering hardware described in Bare Metal [Requirements]({{< relref "./bare-prereq" >}}), you need to prepare the hardware and create a CSV file describing that hardware.
+
+## Prepare hardware
+To prepare your computer hardware for EKS Anywhere, you need to connect your computer hardware and do some configuration.
+Once the hardware is in place, you need to:
+
+* Obtain IP addresses to assign to the IPMI and regular NIC interface on each machine in the cluster.
+* Obtain the gateway address for your network to reach the Internet.
+* Obtain the IP address for your DNS servers.
+* Make sure the following settings are in place:
+  * UEFI is enabled on all target cluster machines
+  * PXE boot is enabled on each NIC
```

Comment:
They only need PXE enabled on the NIC whose IP and MAC addresses they provide in the first step. I will let you phrase that nicely

Reply:
Done.

Reply:
Done.